newIDE/app/src/UI/Theme/DarkTheme/EventsSheet.css

```diff
@@ -125,6 +125,15 @@
   color: #e8dc59;
 }
 
+.gd-events-sheet-dark-theme .name-and-icon-container:empty,
```

Comment:
Can we double check if we need the `name-and-icon-container` rule?
From what I understand you added it because otherwise the background was not showing for objects (and variables?

In theory you should not need to do it on both, because name-and-icon-container is, at the name implies, just a wrapper around a text and an icon. It might be used later for other things totally unrelated to parameters. See the generated markup:

![image](https://user-images.githubusercontent.com/1280130/73690362-13559200-46c8-11ea-8274-69aa8c7caaf4.png)

Reply:
I had to add it in order to cover for object selectors that were empty. Remember this css rule only targets spans of that class with nothing in them

Reply:
I mean, as shown on my screenshot, instruction-parameter is already there - so there should not be a need for adding the same things to name-and-icon-container? It's duplicating style it seems.

Reply:
Try removiny it :)  in the case of objects it's not empty, it's child is empty